R2API/ModlistAPI.cs

```diff
@@ -163,8 +166,8 @@ public static class ModListAPI {
             }
         }
 
-        private static IEnumerator MessageWaitClient( NetworkConnection conn, float duration ) {
-            yield return new WaitForSeconds( duration );
+        private static async void MessageWaitClientAsync( NetworkConnection conn, float duration ) {
```

Comment:
This should be async Task instead of void

Reply:
Do we want to use the unity convention or the C# convention? because as far as I can tell they differ here.

Reply:
In that case definitely Unity, shoulda looked into it first.